client/app/bundles/course/assessment/pages/AssessmentStatistics/AnswerDetails/TextResponseDetails.tsx

```diff
@@ -14,7 +14,7 @@ const TextResponseDetails = (
         dangerouslySetInnerHTML={{ __html: answer.fields.answer_text }}
         variant="body2"
       />
-      {question.allowAttachment && (
+      {question.maxAttachments > 0 && (
```

Comment:
This should have been squashed with 2nd previous commit

Reply:
done